frontend/src/pages/CoursePage/CoursePage.jsx

```diff
@@ -36,6 +36,9 @@ export const CoursePage = () => {
     <>
       <Navigation />
       <h1 className={styles.heading}>{courseData[courseId].name}</h1>
+      <button onClick={() => setTreeObj(treeObj.children[1])}>
```

Comment:
Please remove this. This was probably there when you created your branch but it's no longer needed.

Reply:
Can just do a local merge in order to update branch with local changes no?